scripts/utils/trading_strategy_helpers.js

```diff
@@ -185,8 +145,10 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
 
   /**
    * Retrieves token information foll all tokens involved in the deposits.
+   *
    * @param {(Deposit|Withdrawal)[]} flux List of {@link Deposit} or {@link Withdrawal}
-   * @return {Promise<TokenObject>[]} list of detailed/relevant token information
+   * @param {boolean} [debug=false] prints log statements when true
+   * @returns {Promise<TokenObject>[]} list of detailed/relevant token information
```

Comment:
Yet again, same issue as before.
```suggestion
   * @returns {object} object mapping token addresses to a promise of relevant token information
```